netlify.toml

```diff
@@ -0,0 +1,10 @@
+[build]
+  base = "site/"
+  command = "hugo"
+  publish = "../publishedSite"
+
+[context.production.environment]
+  HUGO_VERSION = "0.92.2"
+
+[context.deploy-preview.environment]
+  HUGO_VERSION = "0.92.2"
```

Comment:
Looks like we could use `[build.environment]` to set this for all envs? https://docs.netlify.com/integrations/frameworks/hugo/#hugo-version